bb_wrapper/services/pix.py

```diff
@@ -37,6 +38,7 @@ def identify_key_type(self, key: str):
         # 4
         try:
             is_uuid = uuid.UUID(key)
+            is_uuid = True
```

Comment:
Isso pode ser feito utilizando um cast para bool do UUID!

```suggestion
            is_uuid = bool(uuid.UUID(key))
```